src/mapDataManipulation/mapDataManipulationUtils.ts

```diff
@@ -24,6 +35,27 @@ export function changeRgbMappingArraysWithFunction(
 // [0,255] to [0,1]: a = 0, b = 255; c = 0, d = 1
 // [0,1] to [0,255]: a = 0, b = 1; c = 0, d = 255
 
+export function changeRgbMappingArrayInterval(
+  rgbMappingArray: number[],
+  oldMin: number,
+  oldMax: number,
+  newMin: number,
+  newMax: number,
+  strictlyLimitValuesToInterval: boolean = false,
```

Comment:
I think the term commonly used is `cap`, so this could be `capValuesToRange`.

Also, there is some inconsistency about the naming - `range` and `interval` are used interchangeably. I would suggest using just one of the terms (imho `range` is more fitting).

Neither of the issues are very important though, as it is clear enough from the context what is happening - fftr either way.

**EDIT:** actually, I think the parameter `strictlyLimitValuesToInterval` is not even needed, as this is something that happens always. Fftr either way.

Reply:
Thanks, true, `capValuesToRange` would be more appropriate. I decided to not use that parameter.